assets/js/googlesitekit-idea-hub-post-list.js

```diff
@@ -47,3 +52,20 @@ domReady( () => {
 		}
 	} );
 } );
+
+domReady( () => {
+	const renderTarget = document.getElementById(
+		'js-googlesitekit-post-list'
+	);
+
+	if ( renderTarget ) {
+		render(
+			<Root
+				viewContext={ VIEW_CONTEXT_POSTS_LIST }
+				withPermissionsModal={ false }
+				withSurveys={ false }
```

Comment:
We pull the current survey and show it in the `CurrentSurveyPortal` component which will not be rendered if `withSurveys` equals `false`. So, answering your question, it will prevent the added survey from showing on this page.

Reply:
My mistake, I was referring to the feature tour 🤦‍♂️ 

As mentioned above, I'm not sure these extra props are needed just yet – for surveys in particular they wouldn't be triggered here so I'm not sure rendering the `CurrentSurvey` would be a problem as there wouldn't be one without being triggered anyways 🙂

Reply:
Ok, removed.